source/fab/language/fortran.py

```diff
@@ -393,5 +392,20 @@ def analyse(self, source: TextReader) -> None:
                         end_values = {'exp': exp[0],
                                       'name': exp[1],
                                       'found': end_name}
-                        raise AnalysisException(
+                        raise TaskException(
                             end_message.format(**end_values))
+        return []
+
+
+class FortranPreProcessor(Command):
+    stdout = False
```

Comment:
This is set but I do not believe it is used anywhere.

Reply:
It's used by the `CommandTask` class in its `run` method - if it is true it causes the task to capture the stdout and direct it into the file given by the `output_filename` property

Reply:
Having said that, `False` is the default in the `Command` base class anyway, so I can take it out (apologies if that's what you were getting at to begin with!)